deploy-templates/Taskfile.yml

```
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: [".task.env", ".env"]
+
+vars:
+  DOCKER_COMPOSE: '{{ .TASK_DOCKER_COMPOSE | default "itkdev-docker-compose-server" }}'
+
+tasks:
+
+  build-site:new:
+    cmds:
+      - "{{.DOCKER_COMPOSE}} up --detach"
```

Here a `compose` task will come in handy:

```suggestion
      - task compose -- up --detach
```